common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -392,44 +449,47 @@ innovation_sentinel = {
 	flag = global_maa
 }
 
-innovation_snow_hunter = {
+innovation_silithid_colossus = {
```

Comment:
Not existing unit type.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/155878539-fcd9106a-fe28-4665-9a20-56eecef3b727.png)

</details>